controllers/embedder_controller.go

```diff
@@ -200,15 +200,18 @@ func (r *EmbedderReconciler) UpdateStatus(ctx context.Context, instance *arcadia
 		if !ok {
 			msg = _StatusNilResponse
 		}
-		// Set status to available
 		newCondition = arcadiav1alpha1.Condition{
 			Type:               arcadiav1alpha1.TypeReady,
 			Status:             corev1.ConditionTrue,
 			Reason:             arcadiav1alpha1.ReasonAvailable,
 			Message:            msg,
-			LastTransitionTime: metav1.Now(),
 			LastSuccessfulTime: metav1.Now(),
 		}
+		if instanceCopy.Status.GetCondition(arcadiav1alpha1.TypeReady).Status != corev1.ConditionTrue {
```

Comment:
we have a IsReady() function https://github.com/kubeagi/arcadia/blob/main/api/v1alpha1/condition.go#L214